functional_tests/features/bundle_ui.feature

```diff
@@ -0,0 +1,267 @@
+Feature: UI Bundle Happy Paths
+
+#---- Bundle Create UI Tests -----
+    Scenario: A Publishing Admin creates a bundle
+        Given there is a Publishing Admin user
+        And there are 3 Preview teams
+        And the user is a member of the Preview teams
+        And there are 3 release calendar pages
+        And there are 3 Statistical Analysis pages
+        And there are 3 bundles
+        When the Publishing Admin logs in
+        And the user can see the Bundles menu item
+        And the user clicks the Bundles menu item
+        Then the user can search for unknown bundle with response "There are no bundles to"
```

Comment:
Should this also say "There are no bundles to display" like on line 39? If it can say the same thing, then we don't need to parameterise our step as well.